temba/flows/views.py

```diff
@@ -927,7 +927,7 @@ def __init__(self, user, *args, **kwargs):
                 ).order_by("created_on")
 
                 keyword_triggers = self.fields["keyword_triggers"]
-                keyword_triggers.initial = ",".join(t.keyword for t in flow_triggers)
+                keyword_triggers.initial = list([t.keyword for t in flow_triggers])
```

Comment:
@ihor-palii Can you please create a new PR for that change and go back to the original here? Because we will only merge to dev.ccl the fixes for v3 for now, after the v3 release, we will merge the other ones. Thank you.